### ISSUE TITLE
R code doesn't work with new WoS data

### ISSUE THREAD

**Issue by breakawaybilly** [aknutas/nails#17]
The cleaning2.R code doesn't work with new WoS data. 
![nailserror](https://cloud.githubusercontent.com/assets/29048870/26561489/d13d04a4-44b6-11e7-80c0-32f85c2118e8.png)

**Comment by aknutas** [comment]
Thanks for reporting! Can you also attach some WoS downloads as samples?

**Comment by breakawaybilly** [comment]
[savedrecs (40).txt](https://github.com/aknutas/nails/files/1036886/savedrecs.40.txt)
[Uploading savedrecs (39).txt…]()

**Comment by breakawaybilly** [comment]
[savedrecs.40.txt](https://github.com/aknutas/nails/files/1036888/savedrecs.40.txt)
[savedrecs (39).txt](https://github.com/aknutas/nails/files/1036891/savedrecs.39.txt)

**Comment by jsalminen** [comment]
Hi @breakawaybilly ,

And thanks for the feedback! It's nice to see that people are  using nails!

I think this issue was fixed in April 29 commit. The code you posted is from an older version. Try downloading the newest version of cleaning2.R, and let us know if there's still an issue. At least for me both example files worked. 

However, there was another little bug in year table merging in exploration.Rmd caused by our reference data for yearly publication volumes being out of date. I fixed that and report does not crash anymore if all literature is newer than 2015. @aknutas , we should still update analyze.csv with fresh publication volumes.
